kernel/packages/decentraland-ecs/types/@decentraland/QuestsController/index.d.ts

```diff
@@ -0,0 +1,107 @@
+declare module '@decentraland/QuestsController' {
+  export enum ProgressStatus {
```

Comment:
.d.ts files cannot have enums (values). Use instead union types

Reply:
Actually, we can. But it has to be a `const` enum

https://github.com/decentraland/explorer/blob/f7225a16bb6b0cf4338af684a1cc11d2df25e8b2/kernel/packages/decentraland-ecs/types/@decentraland/RestrictedActions/index.d.ts#L23

Reply:
the problem will come in this scenario:

```ts
import { ProgressStatus } from '@decentraland/QuestsController'

log(ProgressStatus.NOT_STARTED) 
// => `Error: cannot read property NOT_STARTED of undefined`
```

Because these modules are merely a proxy of an RPC object. And `ProgressStatus` is not only a type, it is a value.